takeshape/takeshape-starter-ecommerce-kitchen-sink
# Updated instructions and screenshots

> If you don't want to invoke CI, put `[skip ci]` in the commit message

- Added more fleshed-out instructions for setting up the services
- I don't know 100% if the Referral Candy info is correct, but from looking at the schema it seems to not use any auth?
- I don't know if the last Voucherify instruction is correct. In the schema it seems we're only setting the `X-App-Id` to authenticate?

### Test Plan (Steps to test):

1. Read the instructions for typos and logical errors.
1. Ensure no screenshots are broken.
1. Ensure no links are broken.

### Checklist:
